src/jit/codegenarm64.cpp

```diff
@@ -1367,7 +1367,8 @@ void                CodeGen::genEmitGSCookieCheck(bool pushReg)
     getEmitter()->emitIns_R_R(INS_cmp, EA_PTRSIZE, regGSConst, regGSValue);
 
     BasicBlock  *gsCheckBlk = genCreateTempLabel();
-    inst_JMP(genJumpKindForOper(GT_EQ, true), gsCheckBlk);
+    emitJumpKind jmpEqual = genJumpKindForOper(GT_EQ, CK_SIGNED);
```

Comment:
Yes, It never matters for Equality comparisons
I decided to always use Signed for every equality OR inequality branch in the codebase.